pages/partials/questionAwardedPoints.ejs

```diff
@@ -0,0 +1,11 @@
+<% include('pointsFormatter'); %>
+
+<% for (var i = 0; i < awarded_points.length; i++) { %>
```

Comment:
If there is nothing to display can we display an `&mdash;` instead of nothing?

Reply:
Sure, easy.

Reply:
Sigh. I think this is the cause of the bug you just fixed.

Reply:
I'm not sure, haven't worked it all out yet

Reply:
We've agreed to defer this issue.